Scripts/EllosBoonSelectorMod.lua

```diff
@@ -384,6 +424,14 @@ function OpenSeedControlScreen( args )
 end
 
 function DoesRewardMatchFilters(roomReward)
+  if EllosBoonSelectorMod.ChaosFilter == "No Chaos" and (roomReward.FirstRoomChaos or roomReward.SecondRoomChaos) then
+    return false
+  elseif EllosBoonSelectorMod.ChaosFilter == "Room 1 Chaos" and not roomReward.FirstRoomChaos then
+    return false
+  elseif EllosBoonSelectorMod.ChaosFilter == "Room 2 Chaos" and not roomReward.SecondRoomChaos then
+    return false
+  end
```

Comment:
Technically this does not support the original functionality of "show if either, but don't filter".  I guess my Hammer search is even more janky, but wanted to call it out

Reply:
I'm not sure what you mean. If no filter is selected in the UI, EllosBoonSelectorMod.ChaosFilter is set to "", so none of these cases will apply and no filtering will happen.